updates/1/updates.sql

```diff
@@ -57,4 +57,52 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xA15C7Ebe1f07C
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64", "C", "OptionRoom Token", "ROOM", 1611783103, NULL, "option-room", "", "0xAd4f86a25bbc20FfB751f2FAC312A0B4d8F88c64");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1", "C", "Tranche Finance", "SLICE", 1609247971, NULL, "tranche-finance", "SLICE", "0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1");
+*
+UPDATE assets SET swapped_for="_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1" WHERE identifier="_ceth_0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7";/*Introduce Tranche Finance token swap*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7", "C", "Jibrel Network Token", "JNT", 1552845272, '_ceth_0x0AeE8703D34DD9aE107386d3eFF22AE75Dd616D1', "jibrel", "JNT", "0xa5Fd1A791C4dfcaacC963D4F73c6Ae5824149eA7");
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x9B02dD390a603Add5c07f9fd9175b7DABE8D63B7", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x9B02dD390a603Add5c07f9fd9175b7DABE8D63B7", "C", "Shopping.io", "SPI", 1607010551, NULL, "shopping-io", "SPI", "0x9B02dD390a603Add5c07f9fd9175b7DABE8D63B7");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x74232704659ef37c08995e386A2E26cc27a8d7B1", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x74232704659ef37c08995e386A2E26cc27a8d7B1", "C", "Strike Token", "STRK", 1615435564, NULL, "strike", "STRK", "0x74232704659ef37c08995e386A2E26cc27a8d7B1");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xE748269494e76c1ceC3F627bb1e561E607dA9161", 8, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xE748269494e76c1ceC3F627bb1e561E607dA9161", "C", "Xels Token", "XELS", 1613731292, NULL, "xels", "XELS", "0xE748269494e76c1ceC3F627bb1e561E607dA9161");
```

Comment:
This contract is not verified in etherscan. Where did you get the data?

Reply:
I got the information from the token view https://etherscan.io/token/0xE748269494e76c1ceC3F627bb1e561E607dA9161

Reply:
nice. Did not notice that. But I guess the decimals is what you got from there only. Not name/symbol. But I guess decimals is most important. Name symbol we can just assume coingecko's one.

(I suppose you can actually use the erc20info token endpoint you made and get them too. Contract not being verified does not mean it does not have the methods)